src/screens/Interests.tsx

```diff
@@ -42,9 +48,43 @@ function Interests() {
 
   };
 
+  useEffect(() => {
+    // fetches resource-lists to use the category
+    const fetchResourceData = async () => {
+      try {
+        const res = await fetch(APIUrl + "resource-lists");
+        const json = await res.json();
+        let jsonData = json.data.map((x: any) => {
+          return {
+            category: x.attributes.category,
+            sub_category: x.attributes.sub_category,
+            link: x.attributes.link,
+          };
+        });
```

Comment:
Since we're not verifying the structure of `x` we should just tell TS what we expect it to be and then you don't have to manually assign things. Ideally, we verify data from the API but....that's a future problem.